<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resultsWindow</class>
 <widget class="QMainWindow" name="resultsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analysis results</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="results_ResultsBreakdown">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>311</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Click on a sentence in the list to show a breakdown of its emotion here.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="results_SaveBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>490</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Save the results of the sentiment analysis as a text file.</string>
    </property>
    <property name="text">
     <string>Save results</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="results_StartOverBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>540</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Delete the results of the analysis &amp; start over with a new piece of text.</string>
    </property>
    <property name="text">
     <string>Start over</string>
    </property>
   </widget>
   <widget class="QListWidget" name="results_AnalysisResults">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>451</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
